include/faabric/scheduler/MpiWorld.h

```diff
@@ -224,15 +225,23 @@ class MpiWorld
 
     std::unordered_map<std::string, std::shared_ptr<InMemoryMpiQueue>>
       localQueueMap;
+
+    // Thread pool for asynchronous work
     std::shared_ptr<faabric::scheduler::MpiAsyncThreadPool> threadPool;
+    int getMpiThreadPoolSize();
```

Comment:
The big question is, can we get rid of the MPI thread pool once we have this stuff in place?